app/Models/Bible/BibleBook.php

```diff
@@ -119,4 +137,51 @@ public function book()
     {
         return $this->belongsTo(Book::class);
     }
+
+    /**
+     * Get collection of bible books sorted by bible versification or the book_seq column if it not empty.
+     *
+     * @param string $bible_id
+     * @param string $book_id
+     * @param string $bible_versification
+     *
+     * @return Collection
+     */
+    public static function getAllSortedByversificationOrBookSeq(
```

Comment:
change method name to getAllSortedByBookSeqOrVersification, to indicate that BookSeq is default/preferred path

Reply:
@bradflood, The method has been updated to: `getAllSortedByBookSeqOrVersification`